docs/codeql/codeql-cli/about-codeql-workspaces.rst

```diff
@@ -30,6 +31,10 @@ For example, the following ``codeql-workspace.yml`` file defines a workspace tha
     ignore:
       - "*/codeql-packs/**/experimental/**/qlpack.yml"
 
+    registries:
+     - packages: '*'
+       url: https://containers.GHE_HOSTNAME/v2/
+
```

Comment:
I expect a lot of customers will want to use `codeql/*` packs from the Dotcom registry and other packs from their GHES instance.  How about using that as an example?

Reply:
sure